pennylane/devices/default_qubit.py

```diff
@@ -77,6 +77,49 @@
     from pennylane.operation import Operator
 
 
+# Hardcoded gate_set from default_qubit.toml
+ALL_DQ_GATE_SET = {
```

Comment:
Don't forget to include the `C(...)` and the `Adjoint(...)`.

Reply:
@astralcai Would it be sufficient if we write other two helpers here to automatically generate the name strings `C(...)` and `Adjoint(...)`? Also curious if there's already tool I can abuse here in the graph decomposition module

Reply:
Yeah that should be fine.

Reply:
I'll continue the conversation in the updated code